uniflow/flow/config.py

```diff
@@ -240,6 +235,49 @@ class RaterClassificationOpenAIGPT4Config(RaterConfig):
     )
 
 
+@dataclass
+class RaterClassificationBedrockClaudeConfig(RaterConfig):
```

Comment:
pull the latest code and rename consistently for the rest called `RaterForClassificationBedrockClaudeConfig`